source/YugiAcademy/YuGiOh/templates/YuGiOh/index.html

```diff
@@ -21,11 +21,18 @@
       id="backgroundVideo"
       onclick="location.href='login';"
     >
-      <source src="../../static/assets/background-video.mp4" type="video/mp4" />
+      <source src="../../static/assets/background-video-yugi.mp4" type="video/mp4" />
     </video>
 
     <div class="content">
-      <h1>Welcome to Yugi Academy</h1>
+      <h1 id="welcome-video-message">Welcome to Yugi Academy</h1>
     </div>
+
```

Comment:
We should add a comment, linking to stack overflow, in order to explain why we are doing this!